src/wallet/rpc/coins.cpp

```diff
@@ -521,6 +521,9 @@ RPCHelpMan listunspent()
                             {"minimumSumAmount", RPCArg::Type::AMOUNT, RPCArg::DefaultHint{"unlimited"}, "Minimum sum value of all UTXOs in " + CURRENCY_UNIT + ""},
                         },
                         "query_options"},
+                    {
+                        "include_immature_coinbase", RPCArg::Type::BOOL, RPCArg::Default{false}, "Include coinbase UTXOs that haven't met the minimum spendability depth requirement\n"
```

Comment:
Please don't add booleans as positional arguments. This should probably be part of `query_options` above.

Reply:
sure thanks, that is better.